optimum/habana/transformers/models/opt/modeling_opt.py

```diff
@@ -107,9 +107,15 @@ def gaudi_opt_attention_forward(
     key_states = key_states.view(*proj_shape)
     value_states = value_states.view(*proj_shape)
 
-    attn_weights = torch.bmm(query_states, key_states.transpose(1, 2)) * self.scaling
+    src_len = key_states.size(1)
+    attn_weights = torch.bmm(query_states, key_states.transpose(1, 2))
```

Comment:
Can we really safely remove scaling? Original transformers code still have it: https://github.com/huggingface/transformers/blob/main/src/transformers/models/opt/modeling_opt.py#L160